feflow/tests/test_nonequilibrium_cycling.py

```diff
@@ -232,6 +232,7 @@ def test_terminal_units(self, protocol_dag_result):
     #             )
 
     @pytest.mark.gpu_ci
+    @pytest.mark.slow
```

Comment:
We were already running these tests in CI before, so they aren't THAT slow. We should not mark them, imo
```suggestion
```